client/src/components/form/Input.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "@emotion/styled";
+
+const Input = ({ topic, type, value, onChange, placeholder }) => {
+  return (
+    <InputContainer>
+      <p>{topic}</p>
+      <input
+        type="text"
+        value={value}
+        onChange={onChange}
+        placeholder={placeholder}
+      />
+    </InputContainer>
+  );
+};
+export default Input;
+
+const InputContainer = styled.label`
+  display: grid;
```

Comment:
Is the label really a container? otherwise just name it StyledInput or something like that